src/memspaces/memspace_lowest_latency.c

```diff
@@ -0,0 +1,103 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <assert.h>
+#include <ctype.h>
+#include <hwloc.h>
+#include <stdlib.h>
+
+#include "base_alloc_global.h"
+#include "memory_target_numa.h"
+#include "memspace_internal.h"
+#include "memspace_numa.h"
+#include "topology.h"
+#include "utils_common.h"
+#include "utils_concurrency.h"
+#include "utils_log.h"
+
+static umf_result_t getBestLatencyTarget(umf_memory_target_handle_t initiator,
+                                         umf_memory_target_handle_t *nodes,
+                                         size_t numNodes,
+                                         umf_memory_target_handle_t *target) {
+    size_t bestNodeIdx = 0;
+    size_t bestLatency = 0;
```

Comment:
I suppose that no node will have a latency less than 0 ;-)

Reply:
I forgot about the init value :|, should be fine now.

Reply:
Thanks to you I've also noticed a bug in NUMA memory target that should return the worst possible attribute value when we wanted to skip the query (when the nodes aren't local).